frontend_tests/zjsunit/zjquery.js

```diff
@@ -365,6 +365,8 @@ exports.make_new_elem = function (selector, opts) {
 
     self.selector = selector;
 
+    self.length = 1;
```

Comment:
It might be worth doing some basic testing of this behavior as well (just an `assert` on `.length` somewhere in the test suite)?

Reply:
I've added a test for this. Thanks!